database/redis/triggers_search_results.go

```diff
@@ -0,0 +1,71 @@
+package redis
+
+import (
+	"fmt"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/database/redis/reply"
+)
+
+const triggersSearchResultsExpireTime = 1800
```

Comment:
1800 what? Maybe rename constant to `triggersSearchResultsExpireTimeSeconds`. Or leave a comment.

Reply:
Fixed